content/lessons-learned-from-the-rust-vision-doc-process.md

```diff
@@ -0,0 +1,103 @@
+
+[vds]: https://blog.rust-lang.org/2025/04/04/vision-doc-survey/
+
+This is the first of a series of blog posts covering what we learned throughout that process and what recommendations we have to offer as a result. This first post is going to go broad. We'll discuss the process we used and where we think it could be improved going forward. We'll talk about some of the big themes we heard -- some that were surprising and others that were, well, not surprising at all. Finally, we'll close with some recommendations for how the project might do more work like this in the future.
+
+[^notaswide]: "As wide a variety of Rust users **as we could find**" -- the last part is important. One of the weaknesses of this work is that we wanted to hear from more Rust skeptics than we did.
+
+## The questions we were trying to answer
+
+One of the first things we did in starting out with the vision doc was to meet with a User Research expert, [Holly Ellis](https://www.linkedin.com/in/holly-ellis-0a336445/), who gave us a quick tutorial on how User Research works[^tyh]. Working with her, we laid out a set of **research questions** that we wanted to answer. Our first cut was very broad, covering three themes:
+
+[^tyh]: Thanks Holly! We are ever in your debt.
+
+* Rust the technology:
+    * "How does Rust fit into the overall language landscape? What is Rust's mission?"
+    * "What brings people to Rust and why do they choose to use it for a particular problem...?"
+    * "What would help Rust to succeed in these domains...?" (e.g., network systems, embedded)
+    * "How can we scale Rust to industry-wide adoption? And how can we ensure that, as we do so, we continue to have a happy, joyful open-source community?"
+* Rust the global project:
+    * "How can we improve the experience of using Rust for people across the globe?"
+    * "How can we improve the experience of contributing to and maintaining Rust for people across the globe?"
+* Rust the open-source project:
+    * "How can we tap into the knowledge, experience, and enthusiasm of a growing Rust userbase to improve Rust?"
+    * "How can we ensure that individual or volunteer Rust maintainers are well-supported?"
+    * "What is the right model for Foundation-project interaction?"
+
+## Step 1: Broad-based survey
+
+Before embarking on individual interviews, we wanted to get a broad snapshot of Rust usage. We also wanted to find a base of people that we could talk to. We created a survey that asked a few short "demographic" questions -- e.g., where does the respondent live, what domains do they work on, how would they rate their experience -- and some open-ended questions about their journey to Rust, what kind of projects they feel are a good fit for Rust, what they found challenging when learning, etc. It also asked for (optional) contact information.
```

Comment:
> how would they rate their experience

this is a bit ambiguous: I think you mean comfort level with Rust, but this could also mean "On a scale from 1-5, how much have you liked your experience with Rust?" (which it isn't).

Reply:
What was the exact question?

Reply:
"How comfortable do you feel writing code in Rust?"